core/domain/user_jobs_continuous.py

```diff
@@ -201,6 +206,7 @@ def _get_most_recent_activity_commits(
 
     @classmethod
     def entity_classes_to_map_over(cls):
+        """Return a list of datastore class references to map over."""
```

Comment:
Here and below: use "Returns" everywhere. There's currently a mix of "Return" and "Returns".

Reply:
done.